homeassistant/helpers/system_info.py

```diff
@@ -15,16 +15,25 @@
 from homeassistant.util.package import is_docker_env, is_virtual_env
 
 from .importlib import async_import_module
+from .singleton import singleton
 
 _LOGGER = logging.getLogger(__name__)
 
+_DATA_MAC_VER = "system_info_mac_ver"
+
 
 @cache
 def is_official_image() -> bool:
     """Return True if Home Assistant is running in an official container."""
     return os.path.isfile("/OFFICIAL_IMAGE")
 
 
+@singleton(_DATA_MAC_VER)
+async def get_mac_ver(hass: HomeAssistant) -> str:
```

Comment:
```suggestion
async def async_get_mac_ver(hass: HomeAssistant) -> str:
```

Shouldn't this be prefixed with `async_`?